### PR TITLE
NTR - Author update

### PR DESCRIPTION
**Pull request by Yunarito**
- added a developer

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,13 @@
...
             "email": "xpand.4beatz@gmail.com",
             "homepage": "https://xpand4b.de",
-            "role": "Developer"
+            "role": "Lead-Developer"
+        },
+        {
+            "name": "Stuart Lux",
+            "email": "stulux98@gmx.de",
+            "homepage": "https://yunarito.de",
+            "role": "Frontend-Developer"
```
Comment (efalsett) [inline composer.json:19]: Are u sure about tis?

Comment (Yunarito) [inline composer.json:19]: dat boi oh shit waddup

Comment (Yunarito) [reply composer.json:19]: yes

Comment (Yunarito) [reply composer.json:19]: yes

**Review by xPand4B — APPROVED** [review]

**Comment by Yunarito** [comment]
@xPand4B Moin Servus Moin

**Pull request merged**